fli-iam/shanoir-ng
# shanoir-issue#2248: study's history

How to test:
Open a study and its "History" tab
Make an import into the study
With another user, make an access request to this study and grant it from a study admin

Should see something like this:
![image](https://github.com/fli-iam/shanoir-ng/assets/117283961/0ead6e3c-eeb1-410a-8db3-14d96de46a72)


Some usernames are empty (92233740 in db), probably because some calls where done throught rabbitmq, which change authentication data. I searched the username from the userId and there are no username for this id.
